test/lib/mayaUsd/render/vp2RenderDelegate/testVP2RenderDelegatePerInstanceInheritedData.py

```diff
@@ -139,22 +142,28 @@ def testPerInstanceInheritedDataPartialOverride(self):
         self._StartTest('inheritedDisplayColor_pxrSurface')
 
     def testPerInstanceInheriedDataBasisCurves(self):
-        self._StartTest('basisCurveInstance')
+        imageVersion = None
+        if getMayaAPIVersion() <= 2023:
```

Comment:
May I recommend you use the following?
```
import maya.mel
if maya.mel.eval("defaultShaderName") == "standardSurface":
    imageVersion == 'standardSurface'
```
This would be more robust.